src/components/Calculator.js

```diff
@@ -1,54 +1,79 @@
 /* eslint-disable no-unused-vars */
 import React from 'react';
 import Buttons from './Button';
+import calculate from './logics/calculate';
 
 /* eslint-disable class-methods-use-this */
 export default class Calculator extends React.Component {
```

Comment:
As stated on the project requirements: 

![image](https://user-images.githubusercontent.com/57421823/155611440-2bc43303-0d9f-4e1b-80fc-c998999d0bcd.png)


Therefore:

- [ ] Please, implement the use of React lifecycle methods on your project.  

Examples are componentDidMout, componentDidUpdate, componentWillUnmount.

That way, we will move closer to approval 👍